docs/OperatorBestPractices.md

```diff
@@ -1,26 +1,59 @@
-# Operator Best Practices
+# Backing Service Best Practices
 
 ## Introduction
 
 The goals of the Service Binding Operator is to make it easier for
 applications developers to bind applications with needed backing
-services, without having to perform manual configuration of secrets,
-configmaps, etc. and to assist operator providers in promoting and
+services, without having to perform manual configuration of `secrets`,
+`configmaps`, etc. and to assist operator providers in promoting and
 expanding the adoption of their operators.
 
-When a ServiceBindingRequest is created the Service Binding Operator
+When a `ServiceBindingRequest` is created the Service Binding Operator
 collects binding information and shares it with application. The
 Binding Service Operator's controller injects the binding information
-into the application's "DeploymentConfig", "Deployment" or “Replicaset"
-as environment variables via an intermediate Secret called "binding-request".
-The binding also works with Knative services as it works with any "deployment"
-controller which has the podspec defined in the its jsonpath as
+into the application's `DeploymentConfig`, `Deployment` or `Replicaset`
+as environment variables via an intermediate Secret.
+The binding also works with Knative `Services` as it works with any API which has the podspec defined in the its jsonpath as
 "spec.template.spec.containers".
 
 This document provides "best practices" guidelines for the development of
 Operators that manage backing services to be bound together with applications
 by the Service Binding Operator.
 
+## Making a Kubernetes Resource Bindable
```

Comment:
By reading the document, I would say that the focus of the whole doc is to show the best practices how a Kubernets Resource can be consumed by an apllication. How about making this as the main title, for example:

"Best Practices for Making a Kube Resource Bindable"

This Section talks about adding annotations to help SBO to discover interesting data. So how about:

"Declare Bindable Data via Annotation"

Reply:
update the title
https://github.com/redhat-developer/service-binding-operator/pull/618/commits/268279c3d0745a5c4bc460537c96761f0872c724

> "Declare Bindable Data via Annotation"

Annotation is more of a detail, I'd say.